cli/cmd/edges.go

```diff
@@ -246,22 +250,27 @@ func writeEdgesToBuffer(rows []*pb.Edge, w *tabwriter.Writer, options *edgesOpti
 			if len(msg) > maxMsgLength {
 				maxMsgLength = len(msg)
 			}
-
 		}
 	}
+
+	// sorting edgeRows by key for alphabetical listing
+	sort.Slice(edgeRows, func(i, j int) bool {
+		return edgeRows[i].key < edgeRows[j].key
```

Comment:
TIOLI: since this is the only place that `key` is used, you could omit adding it as a field on the struct and just compute it here directly.  This has the advantage of keeping all the sorting logic together in one place.

Reply:
Do you mean something like this? 
```
	sort.Slice(edgeRows, func(i, j int) bool {
		keyI := edgeRows[i].src + edgeRows[i].dst
		keyJ := edgeRows[j].src + edgeRows[j].dst
		return keyI < keyJ
	})
```
This would mean recalculating `keyI` and `keyJ` multiple times but it prevents passing `key` down unnecessarily. 👍

Reply:
Yup!  Unless we expect the size of `edgeRows` to be large, I wouldn't worry about extra calculations